client/src/pages/Result/Result.types.ts

```diff
@@ -0,0 +1,10 @@
+import { Leaderboard } from "../../contexts/quiz.context.types"
+
+export type State = {
+    score?: number;
```

Comment:
why score and leaderboard are optional?

Reply:
because the state that comes from uselocation hook will be of State type or null. Its similar like axios response we have to set it right manually . I will read more about this and find the right solution.